policyengine_us/parameters/gov/states/mo/tax/income/credits/non_refundable.yaml

```diff
@@ -1,7 +1,8 @@
 description: Missouri allows these non-refundable tax credits.
 values:
-  2020-01-01:
-  - mo_wftc
+  2020-01-01: []
```

Comment:
Was this failing TAXSIM validation for 2021?

Reply:
No because [match rate](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/states/mo/tax/income/credits/wftc/eitc_match.yaml) for the credit was set to 2023 and the variable as defined for `formula_2023` -> we consolidated to match model standards